packages/lib/src/vis/shared/ThresholdAdjuster.tsx

```diff
@@ -12,16 +12,12 @@ function ThresholdAdjuster(props: Props) {
 
   useEffect(() => {
     const params = raycaster.params.Points;
```

Comment:
`raycaster.params.Points` is always defined apparently. I'm choosing to trust R3F/Three's types.

Reply:
Ok. May have been changed in an update :shrug: